src/intents.cs

```diff
@@ -1174,6 +1154,25 @@ interface INDoubleResolutionResult {
 
 	[Introduced (PlatformName.iOS, 10, 0)]
 	[Unavailable (PlatformName.MacOSX)]
+	[DisableDefaultCtor]
+	[BaseType (typeof (INIntentResolutionResult))]
+	interface INDateComponentsResolutionResult {
+
+		[Static]
+		[Export ("successWithResolvedDateComponents:")]
+		INDateComponentsResolutionResult GetSuccess (NSDateComponents resolvedDateComponents);
```

Comment:
Suggestion for `GetSuccess`, `GetDisambiguation` and `GetConfirmationRequired`.

What if, for the parameters, we go:

- `resolvedDateComponents` -> `resolved`
- `componentsToDisambiguate ` -> `toDisambiguate`
- `componentsToConfirm` -> `toConfirm`